external/netfx/netfx.depproj

```diff
@@ -2,7 +2,7 @@
 <Project ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003" DefaultTargets="Build">
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
   <PropertyGroup>
-    <IsReferenceAssembly>true</IsReferenceAssembly>
+    <BinPlaceRef>true</BinPlaceRef>
```

Comment:
This doesn't actually conflict.  IsReferenceAssembly still works, but it triggers the "OmitTransitiveCompileReferences" behavior which was trimming out all of NETStandard.Library.  For the TargetingPack packages they are flat so they wouldn't hit this.  We should try to be consistent about how we specify this, but that can be cleanup, not conflict.